### ISSUE TITLE
Need a way for Thorium to "pretend" to be a real Chrome

### ISSUE THREAD

**Issue by Tvangeste** [Alex313031/thorium#291]
**Is your feature request related to a problem? Please describe.**
There are some sites and extensions that insist on being used ONLY with "real" Chrome, and if they detect that a fork of Chrome is used, they refuse to work. I am aware of various Capcha libs doing this, as well as some extensions (1Password, 2FAS).

Essentially, this renders Thorium as less comfortable browser solution that is not properly working in some scenarios.

**Describe the solution you'd like, including relevant patches or source**
A command-line switch to enable Chrome-pretend mode would be enough, I guess.

** Additional Info**
As far as I know, the extensions and libraries use `navigator.userAgentData.brands` API to detect if the real Chrome is used or not.

The real chrome provides:
```
[
    {
        "brand": "Chromium",
        "version": "118"
    },
    {
        "brand": "Google Chrome",
        "version": "118"
    },
    {
        "brand": "Not=A?Brand",
        "version": "99"
    }
]
```
 While the Thorium responds with:
```
[
    {
        "brand": "Chromium",
        "version": "117"
    },
    {
        "brand": "Not;A=Brand",
        "version": "8"
    }
]
```

**Comment by gz83** [comment]
Yes, if there is a way to disguise it, it will solve a lot of problems

**Comment by GrimPixel** [comment]
There is a user agent switcher to use: https://add0n.com/useragent-switcher.html.

**Comment by domeniczz** [comment]
That should be a great feature!

On thorium, my 1password extension just frequently ask me to re-login or enter password, so damn annoying. But on Chrome, the extension id totally depends on 1password desktop app, won't always ask for re-login.

**Comment by Tvangeste** [comment]
@domeniczz , exactly my problem that forced me to file this issue: 1Password extension doesn't connect to the 1Password desktop app, since it insists for the browser to be the "official" one.

**Comment by davidsielert** [comment]
@Tvangeste  1Password has a configuration file that you can change to allow it to work. https://1password.community/discussion/120954/how-the-browser-integration-works

**Comment by domeniczz** [comment]
@davidsielert The discussion is for Linux, I explore a little but fail to find out how to do the same thing in Windows. Do you have any advice? Thanks.

**Comment by davidsielert** [comment]
@domeniczz I believe the other platforms you need to use the beta build which is supposed to have a new area in setting to manage allowed browsers as 1Password no longer wishes to vet chrome forks. 

https://1password.community/discussion/140735/extending-support-for-trusted-web-browsers

**Comment by ffranke** [comment]
@davidsielert that does only seem to work when the App is signed, though…

**Comment by Tvangeste** [comment]
So, this issue with Thorium and 1Password is basically killing my use of Thorium, on multiple platforms (Windows and Linux).

I understand that this is not strictly Thorium problem per se, but still, one important use case is not functional with Thorium for me :(

**Comment by lavavex** [comment]
The user agent change is net even fixing logging into sites like Twitch. Something other than user agent is being used to deny thorium access to some sites.
<img src=https://github.com/Alex313031/thorium/assets/27239435/555633e3-f287-485b-8fa7-6f9479b1c9c7></img>

**Comment by NicolasGB** [comment]
Hi! so I wanted to use 1password in thorium but i cannot make it work.

https://1password.community/discussion/120954/how-the-browser-integration-works I followed this as it has been suggested in the comments of this issue but 1password doesn't seem to recognize thorium either.

I added "thorium-browser" to the list which is the name of the bin but nothing, maybe someone managed to make it work and could give me a tip?

Otherwise hope this issue gets resolved, the work on thorium is excellent but not being able to easily use my password manager is kind of a deal breaker for me :(  

Will use it as soon as it is fixed though :smile:

**Comment by i1u5** [comment]
+1 on this, it would also potentially close #58 #291 #538

I think maybe adding a flag or just a launch parameter/env var would help a lot without affecting anything else, it should not be a requirement to use a user agent switcher extension and even if that were to be the solution, it still does not help with other extensions themselves.

**Comment by megalithic** [comment]
Bumping this for visiblity and consideration, as I believe _many_ are affected.

**Comment by megalithic** [comment]
Any news on this @Alex313031?
